src/Sylius/Behat/Context/Ui/Admin/ManagingProductReviewsContext.php

```diff
@@ -149,9 +149,9 @@ public function iChooseAsItsRating($rating)
     }
 
     /**
-     * @Then I should see the product :productName
+     * @Then I should see the product with name :productName
```

Comment:
I think this change is redundant. It was natural as well as grammatically correct, so I don't see any benefit of that change (only disadvantage, more bytes used 😆 ).